examples/library-example/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@examples/library-example",
-  "version": "5.3.4",
+  "version": "6.0.0-alpha.0",
```

Comment:
Why are some `alpha.0` and others `alpha.1`?

Reply:
Related is why are some pinned and some `^`?

Reply:
@jimmynotjim, So the versioning is all handled by lerna. It only updates a version if a package has been modified. I think the only exception here is if you do a major release. so these patch releases only apply to some.

For the second part we don't pin these because lerna actually manages the versions for all local dependencies as well. It automatically bumps the version to latest when we publish changes. I also had issues when I pinned them a long time ago so I just kept it and let lerna do all the work for these.